src/threads.cpp

```diff
@@ -351,24 +351,24 @@ void MapperClass::OctomappingTask() {
 
         // Publish visualization markers iff at least one node is subscribed to it
         bool pub_obstacles, pub_free, pub_obstacles_inflated, pub_free_inflated;
-        pub_obstacles = (obstacle_marker_pub_.getNumSubscribers() > 0);
-        pub_free = (free_space_marker_pub_.getNumSubscribers() > 0);
+        // pub_obstacles = (obstacle_marker_pub_.getNumSubscribers() > 0);
+        // pub_free = (free_space_marker_pub_.getNumSubscribers() > 0);
         pub_obstacles_inflated = (inflated_obstacle_marker_pub_.getNumSubscribers() > 0);
```

Comment:
only publish inflated obstacles

Reply:
why you don't remove those line that you commented out?

Reply:
The reason I commented out instead of deleting is because we might want to retrieve this back in the future.
I think that if we record our bags on the drone (instead of on the basestation), we might want to have this back in `mapper`